bm-persona/Dining/DiningDetailViewController.swift

```diff
@@ -265,9 +265,21 @@ extension DiningDetailViewController {
         chairImage.heightAnchor.constraint(equalToConstant: 25).isActive = true
         chairImage.contentMode = .scaleAspectFit
         
-        //TODO: use actual capacity
-        capBadge.text = "High"
-        capBadge.backgroundColor = Color.highCapacityTag
+        if let occ = diningHall.occupancy, let status = occ.getOccupancyStatus(date: Date()) {
+            switch status {
+            case OccupancyStatus.high:
+                capBadge.text = "High"
+                capBadge.backgroundColor = Color.highCapacityTag
+            case OccupancyStatus.medium:
+                capBadge.text = "Medium"
+                capBadge.backgroundColor = Color.medCapacityTag
+            case OccupancyStatus.low:
+                capBadge.text = "Low"
+                capBadge.backgroundColor = Color.lowCapacityTag
+            }
+        } else {
+            capBadge.isHidden = true
```

Comment:
Should we hide the dining table icon as well?

Reply:
thought about that as well, we should probably establish what we're doing when stuff is unavailable in general. do we want to hide it, say n/a or unknown, rearrange other elements, etc. should be uniform throughout the app

Reply:
Yeah I guess for now we can just leave it